python_modules/libraries/dagster-sdf/dagster_sdf/asset_utils.py

```diff
@@ -36,7 +36,7 @@ def default_asset_check_key_fn(catalog: str, schema: str, table: str) -> AssetCh
     else:
         asset_key = AssetKey([catalog, schema, table])
     return AssetCheckKey(
-        name=f"{catalog}.{schema}.{table}",
+        name="test",
```

Comment:
Is this intentional?

Reply:
Nevermind, I see where this is used further down.

Reply:
@akbog does this not cause issues in the check having a unique identifier for the table?

Reply:
It does not! There's always a single check per table, that aggregates all checks. This feature in general needs work, but this does not break it!